src/main/scala/cromwell/engine/backend/runtimeattributes/CromwellRuntimeAttributes.scala

```diff
@@ -17,7 +17,8 @@ import scala.util.Try
 import scalaz.Scalaz._
 import scalaz._
 
-case class CromwellRuntimeAttributes(docker: Option[String],
+case class CromwellRuntimeAttributes(attributes: Map[String, WdlValue],
```

Comment:
You're right, I think eventually we should only have the map. But right now all the validation logic returns scala types that are used as attributes, which are then used everywhere in Cromwell. As this part of the code is likely to be reworked anyway by intel changes I thought it was not worth rewiring the entire runtime attribute thing to only use a Map[String, WdlValue], or at least not now.